Index: TicketWave/adwa/Prova.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/TicketWave/adwa/Prova.java b/TicketWave/adwa/Prova.java
new file mode 100644
--- /dev/null	(date 1705434539885)
+++ b/TicketWave/adwa/Prova.java	(date 1705434539885)
@@ -0,0 +1,4 @@
+package TicketWave.adwa;
+
+public class Prova {
+}
